Core/Dialog/PSPNpSigninDialog.cpp

```diff
@@ -0,0 +1,349 @@
+// Copyright (c) 2012- PPSSPP Project.
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 2.0 or later versions.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License 2.0 for more details.
+
+// A copy of the GPL 2.0 should have been included with the program.
+// If not, see http://www.gnu.org/licenses/
+
+// Official git repository and contact information can be found at
+// https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
+
+#if defined(_WIN32)
+#include "Common/CommonWindows.h"
+#endif
+#include <TimeUtil.h>
```

Comment:
```suggestion
#include "Common/TimeUtil.h"
```

Use `<>` for system headers that come from the OS or external installed libraries, and use `""` for headers that are part of the source code.  `Common` is part of the source code.  In theory, some OS like OS/2 or Windows 12 could have a TimeUtil.h header and `<>` would prefer the system header over the correct one.

-[Unknown]